pkg/diagnose/connectivity/core_endpoint.go

```diff
@@ -19,18 +19,27 @@ import (
 	"strings"
 
 	forwarder "github.com/DataDog/datadog-agent/comp/forwarder/defaultforwarder"
+	logsConfig "github.com/DataDog/datadog-agent/comp/logs/agent/config"
 	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/config/resolver"
 	"github.com/DataDog/datadog-agent/pkg/config/utils"
 	"github.com/DataDog/datadog-agent/pkg/diagnose/diagnosis"
+	logshttp "github.com/DataDog/datadog-agent/pkg/logs/client/http"
 	"github.com/DataDog/datadog-agent/pkg/util/scrubber"
 )
 
 func init() {
 	diagnosis.Register("connectivity-datadog-core-endpoints", diagnose)
 }
 
+func getLogsHTTPEndpoints() (*logsConfig.Endpoints, error) {
+	// For now, a few parameters are hardcoded
+	datadogConfig := config.Datadog
+	logsConfigKey := logsConfig.NewLogsConfigKeys("diagnose_key", datadogConfig)
```

Comment:
```suggestion
	logsConfigKey := logsConfig.NewLogsConfigKeys("logs_config.", datadogConfig)
```
This will make sure it reads the logs config that the user set

Reply:
Changed!